modular_skyrat/modules/customization/modules/clothing/clothing.dm

```diff
@@ -19,6 +19,110 @@
 	var/greyscale_config_worn_taur_paw
 	var/greyscale_config_worn_taur_hoof
 
+	/// has_taur_sprite[icon_state]: Does this icon_state have ANY taur sprite?
+	var/static/list/has_taur_sprite = list()
+
+	/// has_taur_sprite_suit[icon_state]: has_taur_sprite but for /obj/item/clothng/suit
+	var/static/list/has_taur_sprite_suit = list()
```

Comment:
I don't think you need these at all

Reply:
Could probably simplify that, yeah.

Reply:
Actually, if I want to keep the caching system intact, I _would_ need these. Come to think of it, I'm not sure whatever tiny performance gain the cache would give would be worth having too redundant lists filled with every single icon_state in the game.